src/ansys/mapdl/core/cli/hpc.py

```diff
@@ -0,0 +1,340 @@
+# Copyright (C) 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+"""Submit PyHPS jobs to a cluster"""
+
+import logging
+import os
+from typing import Optional, Union
+
+import click
+
+from ansys.mapdl.core.cli import main
+
+logger = logging.getLogger()
+
+
+@main.command(
+    short_help="Submit jobs to an HPC cluster using PyHPS.",
+    help="""
+    Submit jobs to an HPC cluster using PyHPS.
+
+
+Example
+
+$ pymapdl submit my_file_01.py --requirements_file=requirements.txt --shell_file=main.sh --name="my job" --user=user --password=password --url="https://123.456.789.101:3000/hps"
+""",
+)
+@click.argument("main_file")
+@click.option(
+    "--name",
+    default=None,
+    type=str,
+    help="""Name of the PyHPS project to create.""",
+)
+@click.option(
+    "--url",
+    default=None,
+    type=str,
+    help="""URL where the HPS cluster is deployed. For example: "https://myserver:3000/hps" """,
+)
+@click.option(
+    "--user", default=None, type=str, help="Username to login into the HPC cluster."
+)
+@click.option(
+    "--password",
+    default=None,
+    type=str,
+    help="Password for logging into the HPC cluster.",
+)
+@click.option(
+    "--python",
+    default=None,
+    type=float,
+    help="""Python version to use to create the virtual environment and
+run the Python file. Python3 is used by default in the cluster.""",
+)
+@click.option(
+    "--inputs",
+    default=None,
+    type=str,
+    help="""
+Input arguments for the simulation. You can specify several arguments by
+joining them with commas. Thus, strings defined in this way cannot contain
+commas. Only integers, floats and strings are allowed.
```

Comment:
```suggestion
Input arguments for the simulation. Because you can specify several arguments by
joining them with commas, strings defined in this way cannot contain
commas. Only integers, floats, and strings are allowed.
```